readme.md

```diff
@@ -158,7 +160,7 @@ Once installation is working, try creating and requesting resources. Note that t
 
 * **Question:** At least one of these will throw an error. Which one? How can you get around this issue?
 
-* **Your Answer:** 
+* **Your Answer:**  we can add -_id as in Series.find().select('-_id) even though it's been deleted
```

Comment:
You should receive an error when doing `Series.create().find()`. You'll have to find another method to getting the updated data (for example, making another query).